NgxHTML.sublime-syntax

```diff
@@ -564,6 +565,10 @@ contexts:
     - match: \?(?!\.)
       scope: keyword.operator.ternary.ngx
       push: ng-ternary-expression
+    - match: in{{ident_break}}
+      scope: keyword.operator.comparison.ngx
+    - match: (?:typeof|void){{ident_break}}
```

Comment:
didn't know `typeof` works. TIL

Reply:
According to https://github.com/angular/angular/commit/0361c2d81f5d2c56597002f465c00e9b1c4003e4 it is moved away from the "unsupported operators" table together with `void`.